style.css

```diff
@@ -0,0 +1,190 @@
+:root{
+  --f1:  #b1abc3; 
+  --f2:  #5c5470; 
+  --f3:  #352f44; 
+  --f4:  #2a2438; 
+  }
+
+body {
+  font-family: Arial, sans-serif;
+  font-size: 100%;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  height: 100%;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  align-items: center;
+  width: 100%;
+  background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url('./books-images/pexels-abby-chung-1106468.jpg'); 
+  background-size: cover;
+  background-position: center;
+  height: 300px; 
+  color: #fff;
+  text-align: center;
+  margin-bottom: 10px;
+  transition: background-image 1s ease; /* Define transition for background image */
+}
+
+.sortButtons {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  width: 100%;
+  height: 40px;
+  padding: 5px 30px;
+}
+
+button, select, #btnRate{
+  font-size: 0.875em;
+  margin: 10px;
+  padding: 10px 25px;
+  background-color:rgb(255, 255, 255); 
+  color: var(--f2);
+  border: none;
+  border-radius: 20px;
+  cursor: pointer;
+  height: 100%;
+}
+
+#az {
+  border-radius: 20px 0 0 20px ;
+  padding: 10px;
+  margin-right: 0;
+}
+
+#za{
+ border-radius: 0 20px 20px 0;
+ padding: 10px;
+ margin-left: 5px;
+}
+
+#az-za {
+  display: flex;
+  justify-content: center;
+}
+
+main, .main{
+  display: flex;
+  flex: 1;
+  flex-wrap: wrap;
+  flex-direction: column;
+  align-items: center;
+}
+
+
+.card{
+  display: flex;
+  flex-direction: row;
+  border-bottom: 2px solid rgb(215, 213, 213);
+  max-width: 500px;
+  max-height: 600px;
+  padding: 5px;
+  position: relative;
+}
+
+h2, h3, p{
+  margin:5px 0;
+}
+
+.card img{
+  object-fit: cover;
+  width: 130px;
+  height: 220px;
+  margin: 5px 20px 5px 5px;
+  box-shadow: 3px 3px 5px #AAA;
+}
+
+#rate{
+  position: absolute;
+  margin: 10px;
+  font-size: 0.875em;
+  padding: 10px 10px;
+  color: #fff;
+  border-radius: 25px;
+  background-color: var(--f1);
+}
+
+footer {
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  padding: 10px;
+  text-align: center;
+}
+
+.footer-content {
+  color: #fff;
+  background-color: var(--f3);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 10px;
+  font-size: .6em;
+}
```

Comment:
if you want to make the footer sticky, apply these rules to the  `<footer>` element :  
```
footer {
  position: sticky;
  bottom: 0;
  height: 60px;
  color: #fff;
  background-color: var(--f3);
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 0 10px;
  font-size: .6em;
}
```
like I said in my previous comment, keep either the `footer` or the `<div class="footer-content">` and apply the rules to it ✨

Reply:
I did the changes, and it works fine now😊